### PR TITLE
more clear warning message

### PR DESCRIPTION
**Pull request by haesleinhuepf**
Hi all,

I'm just suggesting to adapt a warning message. It took me a moment to understand it. I hope the updated version is more clear.

Best,
Robert

### CODE REVIEW

**Review by toslunar — APPROVED** [review]
LGTM.  The warning message is specific to CuPy's GPU implementation and not in SciPy.

File: cupyx/scipy/ndimage/measurements.py [hunk 1]
```diff
@@ -319,9 +319,10 @@ def variance(input, labels=None, index=None):
                            cupy.float64, cupy.uint32, cupy.uint64,
                            cupy.ulonglong]:
         warnings.warn(
-            'Using the slower implementation as '
-            'cupyx.scipy.ndimage.sum supports int32, float16, '
-            'float32, float64, uint32, uint64 as data types'
+            'Using the slower implementation because the provided '
+            'type is not supported by cupyx.scipy.ndimage.sum. '
```
Comment (emcastillo) [inline cupyx/scipy/ndimage/measurements.py:323]: How about showing here the unsupported type to helpwith debugging :).

**Review by emcastillo — APPROVED** [review]
Thanks!, this LGTM! just a very small nitpick

File: cupyx/scipy/ndimage/measurements.py (hunk 1 above)
Comment (haesleinhuepf) [inline cupyx/scipy/ndimage/measurements.py:323]:
```suggestion
            f'type {input.dtype} is not supported by cupyx.scipy.ndimage.sum. '
```
Like that @emcastillo ?  (-:

**Review by emcastillo — APPROVED** [review]

**Comment by toslunar** [comment]
run jenkins please

**Comment by emcastillo** [comment]
run jenkins please

**Comment by emcastillo** [comment]
Test failures are unrelated, merging manually.

Thanks!

**Pull request merged**